tdrs-frontend/src/reducers/reports.test.js

```diff
@@ -34,6 +40,22 @@ describe('reducers/reports', () => {
     ).toEqual({
       year: '',
       stt: 'florida',
+      quarter: '',
+    })
+  })
+
+  it('should handle "SET_SELECTED_QUARTER"', () => {
+    expect(
+      reducer(undefined, {
+        type: SET_SELECTED_QUARTER,
+        payload: {
+          quarter: 'Q1',
```

Comment:
How big of a lift would it be to make this test iterate through all the choices and assert each one gets set correctly?

Reply:
Not a big lift, I can do implement it here